howtos/how-to-add-a-hidden-user-to-a-talkjs-conversation/README.md

```diff
@@ -0,0 +1,20 @@
+This is an example project to go with our tutorial on how to add a hidden user to your TalkJS chat.
+
+<!-- Add link to tutorial once published -->
+
+This project adds a new [TalkJS Theme](https://talkjs.com/docs/Features/Themes/) and edits different components within it to hide a user that has a custom property.
+
+## Prerequisites
+
+To run this tutorial, you will need:
+
+- A [TalkJS account](https://talkjs.com/dashboard/login)
+- 
+
+## How to run the tutorial
+
+1. Clone or download this project.
+2. Go to your TalkJS dashboard and navigate to the **Themes** tab. Create a new theme called "hidden_user" based on the "default" theme.
+3. Add a custom attribute and the "default" role to all your users in the group chat.
```

Comment:
I'd suggest swapping these two steps to have all the theme-related stuff together

```suggestion
2. Add a custom attribute and the "default" role to all your users in the group chat.
3. Go to your TalkJS dashboard and navigate to the **Themes** tab. Create a new theme called "hidden_user" based on the "default" theme.
```

Reply:
I think adding the custom attribute and default role to all users in the group chat is not needed if they're cloning this project.